go/pkg/bertymessenger/db_test.go

```diff
@@ -28,9 +32,18 @@ func getInMemoryTestDB(t testing.TB, opts ...getInMemoryTestDBOpts) (*dbWrapper,
 	t.Helper()
 
 	init := true
+	log := zap.NewNop()
+
 	for _, o := range opts {
-		if o == getInMemoryTestDBOptsNoInit {
+		switch o {
+		case getInMemoryTestDBOptsNoInit:
 			init = false
+		case getInMemoryTestDBOptsStdOutLogger:
+			var err error
+			log, err = zap.NewDevelopment()
```

Comment:
```suggestion
			log = testutil.Logger(t)
```

to be consistent with the rest of the codebase, or maybe there is a specific reason here?

Reply:
No specific reason here, I hadn't `testutil.Logger` in mind at the moment I wrote those lines